### PR TITLE
Diego Costa - Week 1 

### PR DESCRIPTION
**Pull request by diegocosta-dev**
> ### Tasks
> * [ ]  Create a fork of this repo
> * [ ]  Clone your fork to your local dev environment
> * [ ]  Install the dependencies and run the `npm run dev` command without issues
> * [ ]  Confirm access to the design Figma file
> * [ ]  Review the implementation approach with Lead Developer
> * [ ]  Create website "shell" layout on `_generic_page_layout.twig`
>   
>   * [ ]  Review and define webfonts
>   * [ ]  Review and define colors
>   * [ ]  Review and define containers
>   * [ ]  Implement `_site_header.twig` component
>   * [ ]  Implement `_site_footer.twig` component
>   * [ ]  Implement `typography.twig` page
>   * [ ]  Make sure non-photo images are either inline SVG or URL encoded when used in CSS.

### CODE REVIEW

**Comment by diegocosta-dev** [comment]
> Você pode criar uma classe no arquivo typography.twig: Eu criei com nome "visual", mas pode ser qualquer nomenclatura. image e no arquivo typography.css você faz assim: Nomea com a classe e aí você não precisa adicionar para cada tag uma classe. image

Legal, vou atualizar isso no meu repositório, obrigado @Melnatal

File: src/templates/_partials/_logo_colorful.svg.twig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg width="255" height="101" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (webrgp) [inline src/templates/_partials/_logo_colorful.svg.twig:1]: The _logo_colorful.svg.twig needs to display in two sizes, one for mobile and one for desktop.

File: src/templates/_partials/_site_header.twig [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+								{% if item.text != "Schedule a Call" %}
+									<li><a href="{{item.url}}" class="font-bold text-[#9B51E0] md:text-black md:font-semibold md:text-sm tracking-[2px] leading-[27px] text-[22px] md:py-5">{{item.text}}</a></li>
+									{% else %}
+									<li><a href="#" class="font-bold text-[#9B51E0] 
+									md:font-semibold md:text-sm tracking-[2px] 
+									leading-[27px] text-[22px] md:bg-plum-gradient md:rounded-md 
+								md:text-white md:p-5">Schedule a Call</a></li> 
+								{% endif %}
+							{% endfor %}
+						</ul>
```
Comment (webrgp) [inline src/templates/_partials/_site_header.twig:41]: The styles for desktop are wrong. Check Figma.

File: src/templates/_partials/_site_header.twig [hunk 3]
```diff
@@ -1,3 +1,54 @@
 <header id="site-header">
-  <h2>site-header</h2>
+	<section class="max-w-screen-xl mx-auto pb-10 md:pb-5 md:px-5">
+
+			<div class="bg-plum-lightest md:bg-transparent md:mb-[-30px] overflow-x-scroll scroll md:overflow-x-hidden">
+				<div class="w-max md:w-full mx-auto">
+					<div class="flex gap-4 p-2 md:justify-end">
+						{% for item in headerInfo %}
+							<a href="{{item.url}}" class="uppercase font-medium text-plum z-10">{{item.text}}</a>
```
Comment (webrgp) [inline src/templates/_partials/_site_header.twig:8]: Text style is wrong. Check Figma.

File: src/templates/typography.twig [hunk 4]
```diff
@@ -0,0 +1,73 @@
+{% extends "_layouts/_generic_page_layout.twig" %}
+
+{% block headLinks %}
+	{{ parent() }}
+	<link rel="amphtml" href="/amp">
+{% endblock headLinks %}
+
+{% block content %}
+<div class="typography bg-plum-lightest">
```
Comment (webrgp) [inline src/templates/typography.twig:9]: background color should move to `body`.

**Review by webrgp — CHANGES_REQUESTED** [review]
Good job!

File: src/templates/_partials/_logo_colorful.svg.twig (hunk 1 above)
Comment (diegocosta-dev) [reply src/templates/_partials/_logo_colorful.svg.twig]: Fixed

File: src/templates/_partials/_site_header.twig (hunk 2 above)
Comment (diegocosta-dev) [reply src/templates/_partials/_site_header.twig]: Fixed

File: src/templates/_partials/_site_header.twig (hunk 3 above)
Comment (diegocosta-dev) [reply src/templates/_partials/_site_header.twig]: Fixed

File: src/templates/typography.twig (hunk 4 above)
Comment (diegocosta-dev) [reply src/templates/typography.twig:9]: Fixed

**Pull request closed without merging**